src/util/clipboard.ts

```diff
@@ -1,16 +1,16 @@
 import { logger } from './logger';
-import * as clipboardy from 'clipboardy';
+import * as vscode from 'vscode';
 
 export class Clipboard {
-  public static Copy(text: string) {
+  public static async Copy(text: string): Promise<void> {
     try {
-      clipboardy.writeSync(text);
+      await vscode.env.clipboard.writeText(text);
     } catch (e) {
       logger.error(e, `Clipboard: Error copying to clipboard. err=${e}`);
     }
   }
 
-  public static Paste(): string {
-    return clipboardy.readSync();
+  public static Paste(): Thenable<string> {
```

Comment:
return Promise instead?

Reply:
I actually am not really sure how to properly implement a thenable interface (which the clipboard read is) as a promise. Any help would be much appreciated...

Reply:
Should just be:

```suggestion
  public static async Paste(): Promise<string> {
    return vscode.env.clipboard.readText();
  }
```